spinalcordtoolbox/scripts/sct_detect_pmj.py

```diff
@@ -2,10 +2,17 @@
 
 """Detect Ponto-Medullary Junction.
 
+The models were trained as explained in (Gros et al. 2018, MIA, doi.org/10.1016/j.media.2017.12.001),
+in section 2.1.2, except that the cords are not straightened for the PMJ disc detection task.
+
+To train a new model:
+- Edit "$SCT_DIR/dev/detect_c2c3/config_file.py" according to your needs, then save the file.
+- Run "source sct_launcher" in a terminal
```

Comment:
I don't think `source sct_launcher` is valid anymore, since we no longer have a `bin/sct_launcher`. (Removed in #1996)

If I understand correctly, these lines were copied and pasted from this file?

https://github.com/spinalcordtoolbox/spinalcordtoolbox/blob/67170203934274498b90bc65f6a084f724b6ec19/spinalcordtoolbox/vertebrae/detect_c2c3.py#L4-L17

Reply:
> If I understand correctly, these lines were copied and pasted from this file?

yes this is correct-- sorry i didn't have the time to properly curate the addition-- i preferred to put something rather than to leave it empty-- context: Sandrine was working on a related project and was missing this info

Reply:
It looks like `dev/detect_c2c3/config_file.py` doesn't work out of the box either -- the changes since 2018 have broken it in other ways, too. So, nix my suggestion about updating the steps. :sweat_smile:

Reply:
Apologies for the delay for my suggestion ==> 0910b8d